source/features/latest-tag-button.tsx

```diff
@@ -111,21 +107,27 @@ async function init(): Promise<false | void> {
 		link.append(' ', <span className="css-truncate-target">{latestTag}</span>);
 	}
 
-	if (currentBranch === latestTag || isUpToDate) {
+	if (currentBranch === latestTag || aheadBy === 0) {
 		link.setAttribute('aria-label', 'You’re on the latest release');
 		link.classList.add('disabled', 'tooltipped', 'tooltipped-ne');
 		return;
 	}
 
 	const defaultBranch = await getDefaultBranch();
 	if (currentBranch === defaultBranch) {
-		const aheadBy = await getAheadByCount(latestTag);
-		if (!aheadBy) {
-			return;
-		}
-
-		link.setAttribute('aria-label', `${defaultBranch} is ${aheadBy} commits ahead of the latest release`);
-		link.append(' ', <sup>+{aheadBy}</sup>);
+		const compareLink = (
+			<a
+				className="btn btn-sm btn-outline tooltipped tooltipped-ne"
+				href={`/${getRepoURL()}/compare/${latestTag}...${defaultBranch}`}
+				aria-label={`Compare ${latestTag}...${defaultBranch}`}
+			>
+				<DiffIcon/>
+			</a>
+		);
+
+		link.append(<sup>{` +${aheadBy ? aheadBy : ''}`}</sup>);
+		link.setAttribute('aria-label', `${defaultBranch} is ${aheadBy ? pluralize(aheadBy, '1 commit', '$$ commits') : '20+ commits'} ahead of the latest release`);
```

Comment:
It's **NOT** 20+, it's undetermined. Just because we haven't found the tag in the latest 20 commits of the current branch, it doesn't mean that it's anywhere on the current branch. Some tag the `production` branch while having `develop` as their default
```suggestion
		link.setAttribute('aria-label', aheadBy ? `${defaultBranch} is ${pluralize(aheadBy, '1 commit', '$$ commits')} ahead of the latest release` : `The HEAD of ${defaultBranch} isn’t tagged`);
```

Reply:
You can make that happen by:

1. Creating a branch
2. Making a commit
3. Tagging the commit

You'll see that the API returns the "latest tag", but it can't be matched to any commit on master.

Reply:
Got it. Thanks!